core/src/main/java/io/forestframework/ext/core/AutoRoutingScanExtension.java

```diff
@@ -97,10 +99,10 @@ private boolean isGenericReturnTypeValid(Type genericReturnType) {
         Type rawType = paramType.getRawType();
         Type actualTypeArgument = paramType.getActualTypeArguments()[0];
 
-        return  rawType == io.vertx.core.Future.class && actualTypeArgument == java.lang.Void.class
-                || rawType == java.util.concurrent.CompletableFuture.class && actualTypeArgument == java.lang.Void.class
-                || rawType == io.vertx.core.Future.class && actualTypeArgument == java.lang.Boolean.class
-                || rawType == java.util.concurrent.CompletableFuture.class && actualTypeArgument == java.lang.Boolean.class;
+        return (rawType == Future.class && actualTypeArgument == Void.class)
+                || (rawType == CompletableFuture.class && actualTypeArgument == Void.class)
+                || (rawType == Future.class && actualTypeArgument == Boolean.class)
```

Comment:
`Future` alone can be confusing: is it `java.util.Concurrent.Future` or `io.vertx.core.Future`?